### PR TITLE
TLC-77: Improve chief complaint, add consultation images, remove Para…

### PR DESCRIPTION
**Pull request by ouiliam**
…cetamol 200, add ICD10-Q79, add pregnancy labtest

	modified:   configuration/concepts/concepts.csv
	modified:   configuration/concepts/consultation_form.csv
	modified:   configuration/concepts/diags.csv
	modified:   configuration/drugs/drugs.csv
	modified:   configuration/drugs/drugs_void.csv

### CODE REVIEW

File: configuration/drugs/drugs.csv [hunk 1]
```diff
@@ -55,7 +55,6 @@ ff094ae9-f707-4a4c-befc-e64de5195172,,Metronidazole 250mg,Cambodia_Metronidazole
 88209b1e-5b32-4f39-a5b2-acaaa31ae0ac,,Omeprazole 20mg,Cambodia_Omeprazole,Capsule,20mg,
 165c9b15-5cc7-444d-be87-52ff2c4ccd26,,Oral rehydration 20.5g,Cambodia_Oral rehydration,Suspension,20.5g,
 18d992b2-846c-4ad3-9af8-bbf06b4d6142,,Paracetamol 500mg,Cambodia_Paracetamol,Tablet,500mg,
-a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e,,Paracetamol 200mg,Cambodia_Paracetamol,Tablet,200mg,
```
Comment (rbuisson) [inline configuration/drugs/drugs.csv:58]: @ouiliam in order to void this drug could you use the Void/Retire field directly at this line, in this file?
I mean I see that in "drugs_void.csv" you have added this drug's UUID so what you have done is correct. Just that the "drugs_void.csv" is more for the initial voiding of the default data (that come with the demo).
In this particular case, the drug we void is one of our durgs, let's use the Void/Retire field in this file for that.

File: configuration/concepts/concepts.csv [hunk 2]
```diff
@@ -318,4 +319,9 @@ bb655344-3f10-11e4-adec-0800271c1b75,,SNOMED:258706009,Month(s),ខែ,Month(s),
 9d5b708e-3f10-11e4-adec-0800271c1b75,,,Dosing Units,ឯកតាកំរិត,Dosing Units,ឯកតាកំរិត,,,ConvSet,N/A,,,Unit(s);Capsule(s); Tablet(s); Drop;Teaspoon;  Tablespoon; Puff(s); ml; mg; IU; ,,,,,,,,
 e74aebd1-51b5-424f-b9f0-339fd8cd3588,,,Female,Female,Female,ស្រី,,,Misc,N/A,,,,,,,,,,,
 544634f2-4c08-48b0-9b2b-35251189a59b,,,Male,Male,Male,បុរស,,,Misc,N/A,,,,,,,,,,,
-c46896fd-3f10-11e4-adec-0800271c1b75,,,Patient Document,Patient Document,Patient Document,Patient Document,,,ConvSet,N/A,,,Image;,,,,,,,,
\ No newline at end of file
+c46896fd-3f10-11e4-adec-0800271c1b75,,,Patient Document,Patient Document,Patient Document,Patient Document,,,ConvSet,N/A,,,Image;,,,,,,,,
+,,,Dizziness,Dizziness,Dizziness,,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/concepts.csv:323]: @ouiliam , you did not want to provide translations for those?

File: configuration/drugs/drugs.csv (hunk 1 above)
Comment (ouiliam) [reply configuration/drugs/drugs.csv:58]: @rbuisson OK, have amended the changes, got this error but normal as drug already voided:

ERROR - CsvParser.saveAll(172) |2018-03-19 11:53:23,715| An OpenMRS object could not be constructed or saved from the following CSV line: [a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e, TRUE, Paracetamol 200mg, Cambodia_Paracetamol, Tablet, 200mg, null]
org.openmrs.api.ValidationException: 'Drug[hashCode=eec0d862,uuid=a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e]' failed to validate with reason: concept: An empty concept is not allowed
WARN - HibernateConceptDAO.getConceptByName(1741) |2018-03-19 11:53:23,784| Multiple concepts found for 'Tablet'

File: configuration/concepts/concepts.csv (hunk 2 above)
Comment (rbuisson) [reply configuration/concepts/concepts.csv:323]: @ouiliam Still this question here:
>you did not want to provide translations for those?

File: configuration/drugs/drugs.csv (hunk 1 above)
Comment (rbuisson) [reply configuration/drugs/drugs.csv:58]: OK. I am quite surprise of this. Could you remove the contents of the line and only keep the UUID and Void/Retire fields and try it out?
Something like this:
`a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e,TRUE,Paracetamol 200mg,Cambodia_Paracetamol,Tablet,200mg,`
to become:
`a70fe3db-b6a8-4566-a0c7-c5d8ff6fc67e,TRUE,,,,,`

**Review by rbuisson — APPROVED** [review]

**Comment by rbuisson** [comment]
Merged with a cherry-pick

**Pull request closed without merging**